dxr/app.py

```diff
@@ -162,6 +164,30 @@ def _tree_tuples(trees, tree, query_text, is_case_sensitive):
              description)
             for t, description in trees.iteritems()]
 
+@dxr_blueprint.route('/<tree>/images/<path:path>')
+def send_image(tree, path):
```

Comment:
I'd like to see this merged into browse() somehow. We shouldn't need a separate kind of URL for images (or other things we can display). Just think: what would actually visiting the image at its browse()-style URL do otherwise?